bigtable/google/cloud/bigtable/row.py

```diff
@@ -55,6 +55,13 @@ def __init__(self, row_key, table):
         self._row_key = _to_bytes(row_key)
         self._table = table
 
+    @property
+    def row_key(self):
+        return self._row_key
+
+    @property
+    def table(self):
+        return self._table
```

Comment:
@skyfox We could probably pull these two `@property`-s out into a separate PR since they are not controversial or complex.